CHANGES.md

```diff
@@ -14,6 +14,9 @@ Release Notes.
 * Support collecting logs of log4j, log4j2, and logback in the tracing context with a new `logger-plugin`.
 * Fix the unexpected RunningContext recreation in the Tomcat plugin.
 
+#### Python Agent
+* Added plugin for aiohttp module requester and server.
```

Comment:
This changlog belongs to the OAP-backend. 
And added->add.

Reply:
I am confused, so what do you want me to add and where?

Reply:
> I am confused, so what do you want me to add and where?

Done for you

Reply:
Sorry, review from cell phone, was trying to type less words.
1. Move the change into OAP-backend part, as the file you changed belongs there. 
2. As this
```suggestion
* Add component id for Python aiohttp plugin.
```

The changelog file in the main repo is tracking the its own change. New python plugin belongs to skywalking-python repo's changelog.